From 86f7f2b96a02701b9209d5acb27432fd0e4aad39 Mon Sep 17 00:00:00 2001
From: Joseph Zavodny <Joseph.Zavodny@jhuapl.edu>
Date: Tue, 25 Aug 2015 15:27:57 -0400
Subject: [PATCH 185/793] dma: xilinx: Fix compilers warnings in axidma test
 client.

config should not be cast to an unsigned long.  Its type matches
the function parameters without a cast.

Signed-off-by: Joseph Zavodny <Joseph.Zavodny@jhuapl.edu>
Acked-by: Kedareswara rao Appana <appanad@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/dma/xilinx/axidmatest.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/xilinx/axidmatest.c b/drivers/dma/xilinx/axidmatest.c
index 20cdaf2..2a8af97 100644
--- a/drivers/dma/xilinx/axidmatest.c
+++ b/drivers/dma/xilinx/axidmatest.c
@@ -340,11 +340,11 @@ static int dmatest_slave_func(void *data)
 		/* Only one interrupt */
 		config.coalesc = 1;
 		config.delay = 0;
-		xilinx_dma_channel_set_config(rx_chan, (unsigned long)&config);
+		xilinx_dma_channel_set_config(rx_chan, &config);
 
 		config.coalesc = 1;
 		config.delay = 0;
-		xilinx_dma_channel_set_config(tx_chan, (unsigned long)&config);
+		xilinx_dma_channel_set_config(tx_chan, &config);
 
 		rxd = rx_dev->device_prep_slave_sg(rx_chan, rx_sg, bd_cnt,
 				DMA_DEV_TO_MEM, flags, NULL);
-- 
1.7.5.4

